doc/nrf/samples/samples_zigbee.rst

```diff
@@ -3,6 +3,78 @@
 Zigbee samples
 ##############
 
+The nRF Connect SDK provides the following samples showcasing the :ref:`ug_zigbee` protocol.
+The samples can be built for various boards and can be configured for different usage scenarios.
+
+In addition to their basic functionalities, the samples can be expanded with variants and extensions.
+The following table lists variants and extensions available for each Zigbee sample:
+
+.. list-table::
+    :widths: auto
+    :header-rows: 1
+
+    * - Variant or extension
```

Comment:
What about `Zigbee Shell library`? It can be enabled for the same samples as `Zigbee Endpoint Logger` can.

Reply:
The libraries are documented on the LIbraries for Zigbee page. Here, I'm only listing variants or extensions. If we take the approach of listing the shell library, then other libraries could also be seen as potential extensions.